torch/lib/THCUNN/generic/SpatialFullConvolution.cu

```diff
@@ -2,65 +2,6 @@
 #define THC_GENERIC_FILE "generic/SpatialFullConvolution.cu"
 #else
 
-static inline void THNN_(SpatialFullConvolution_shapeCheck)(
```

Comment:
I am extremely sorry. The blame is on me for this one. The merge was done with older branch and I just copied some cu files over as I got tired and other files being different in just types.